doozer/doozerlib/cli/scan_osh.py

```diff
@@ -33,6 +34,11 @@
 """
 
 
+class ImageType(Enum):
```

Comment:
`BuildType` might be better?  We've also used `kind` to differentiate between image/rpm in some places

Reply:
Fixed in https://github.com/openshift-eng/art-tools/pull/219/commits/dffa83bbf39d2848aa646704b807c3817b6b371e